tests/page-templates/07-autoxhr/52-xhr-response-url.html

```diff
@@ -0,0 +1,42 @@
+<%= header %>
+<!--
+IE9 (and older) and phantomjs do not fire 'loadend' for XHRs, no autoxhr beacon was being sent
```

Comment:
Can you detect those browsers easily?  If they'd "fail" this test, I'd prefer the `numBeacons` be adjusted to what they send and then `this.skip()` the tests so they still "pass".